e2e-tests/admin/pages/targets.js

```diff
@@ -70,32 +70,8 @@ export class TargetsPage extends BaseResourcePage {
    * @param {string} alias alias used for the target
    * @returns Name of the target
    */
-  async createTargetWithAddressAndAlias(
```

Comment:
It looks like this function was split into two separate ones that do nearly the same thing? I think we should start moving towards having helpers that are edition agnostic or just check the actual version we're running against if it can't be determined through the normal inputs, rather than having separate functions since most of the time they're pretty similar. I'm hoping this might also help in writing tests that can be shared between both editions and reduce the split of ENT vs CE tests. 

Could we have made this function work by instead doing a check on the targetType (and removing the default option, since I think TCP is always the default one now) and if it doesn't exist, don't try and fill out that field?